.ruby-version

```diff
@@ -0,0 +1 @@
+2.5.5
```

Comment:
Why ruby 2.5.5 specifically instead of 2.6.4 (or 2.5.6 even).  If we have to use a specific version we should at least use the latest patch I imagine.

I'm fine with locking a version, but just testing it out I had `rbenv` throw me `rbenv: version '2.5.5' is not installed`

Reply:
That is the version that is currently being used in production. If you're using a different version locally, you'll have to install the specified version.